From 7e1accb5a2638c9d9c4d9f7b3e53d6e34351de76 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Fri, 7 Apr 2017 18:42:09 +0000
Subject: [PATCH] OS-6041 regression from illumos#5157 merge Reviewed by:
 Patrick Mooney <patrick.mooney@joyent.com> Reviewed by: Hans Rosenfeld
 <hans.rosenfeld@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 usr/src/tools/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/usr/src/tools/Makefile b/usr/src/tools/Makefile
index 54ac3aba69..36d49d1552 100644
--- a/usr/src/tools/Makefile
+++ b/usr/src/tools/Makefile
@@ -56,6 +56,7 @@ COMMON_SUBDIRS= \
 #  special versions of commands for use only in build
 #
 UNSHIPPED_SUBDIRS = \
+	man \
 	mandoc
 
 sparc_SUBDIRS= \
-- 
2.21.0

